themes/courses.labster.com/lms/templates/emails/unenroll_email_enrolledmessage.txt

```diff
@@ -0,0 +1,29 @@
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+
+% if full_name:
+${_("Dear {full_name}, ").format(full_name=full_name)}
+% else:
+${_("Hi there, ")}
+% endif
+
+${_("You have been un-enrolled in {course_name} at {site_name} by a member "
+    "of the course staff. The course will no longer appear on your "
+    "{site_name} dashboard.").format(
+        course_name=display_name or course.display_name_with_default_escaped, site_name=site_name
+    )}
+
+${_("Your other courses have not been affected.")}
+
+${_("If you have any questions, please contact your Customer Success Specialist or {link_start}contact us{link_end}.").format(
+       link_start=HTML('<a href="{0}">'.format(settings.LABSTER_TECH_SUPPORT_LINK)),
```

Comment:
why not `{{ support_link|safe }}` everywhere? can we make common context?

Reply:
@TamoshaytisV  we try to avoid makes some changes on Edx side and if we implement like your suggestion I think we require additional changes to modify the context in all view.